.gitignore

```diff
@@ -50,3 +50,6 @@ packages/**/lib
 generated
 storybook-static
 tsconfig.tsbuildinfo
+
+# to save thunder https://marketplace.visualstudio.com/items?itemName=rangav.vscode-thunder-client files
+.thunder
```

Comment:
Thunderstruck!

Reply:
Very good tool for rest checks, but I need folder to preserve envs and requests